web/src/main/webapp/WEB-INF/pages/application_state.jsp

```diff
@@ -210,15 +205,17 @@
                           <p class="form-control-static ${stat.first ? 'stateData' : ''}">L${data.recommendedPositionLevel}</p>
                           <c:if test="${stat.first}">
                               <spring:message code="statehistory.error.recommendedPositionLevel.range" var="errorRecommendedPositionLevelRangeMessage"/>
-
                               <select required class="stateInput hidden" id="recommendedPositionLevelInput"
                                 data-bind="valueWithInit: 'recommendedPositionLevel'"
                                 data-error="${errorRecommendedPositionLevelRangeMessage}">
-                                  <option disabled <c:if test="${data.recommendedPositionLevel eq null}"> selected="selected" </c:if>>
-                                                                          <spring:message code="common.pleaseChoose"/></option>
+                                <option disabled ${data.recommendedPositionLevel eq null ? 'selected="selected"' : ''}>
```

Comment:
These fixes belong to the first test state, but they should be implemented for coding state.

I suggest to revert the changes made for the first state.